vm/systemSmartContracts/auction.go

```diff
@@ -97,7 +97,9 @@ func NewStakingAuctionSmartContract(
 
 // Execute calls one of the functions from the auction staking smart contract and runs the code according to the input
 func (s *stakingAuctionSC) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
-	if CheckIfNil(args) != nil {
+	err := CheckIfNil(args)
+	if err != nil {
+		s.eei.AddReturnMessage("nil arguments: error " + err.Error())
```

Comment:
"nil arguments: " + err.Error() is enough

Reply:
done.